learnify/mobile/lib/features/auth/forget_password/view/forget_password_screen.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+import '../../../../core/base/view/base_view.dart';
+import '../../../../core/extensions/context/context_extensions.dart';
+import '../../../../core/extensions/context/theme_extensions.dart';
+import '../../../../core/helpers/selector_helper.dart';
+import '../../../../core/helpers/validators.dart';
+import '../../../../core/managers/navigation/navigation_manager.dart';
+import '../../../../core/widgets/buttons/action_button.dart';
+import '../../../../core/widgets/text-field/custom_text_form_field.dart';
+import '../../../../core/widgets/text/base_text.dart';
+import '../../../../product/constants/icon_keys.dart';
+import '../../../../product/constants/navigation_constants.dart';
+import '../../../../product/language/language_keys.dart';
+import '../constants/widget_keys.dart';
+import '../view-model/forget_password_view_model.dart';
+
+part './components/email_form.dart';
+
+class ForgetPasswordScreen extends BaseView<ForgetPasswordViewModel> {
+  const ForgetPasswordScreen({Key? key})
+      : super(builder: _builder, scrollable: true, key: key);
+  static Widget _builder(BuildContext context) => Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: <Widget>[
+          Image.asset(IconKeys.logo,
+              width: context.width * 33, color: context.primary),
```

Comment:
Remove this color parameter since we changed the logo, it has its color as default.